notifications_utils/field.py

```diff
@@ -49,9 +49,13 @@ def __repr__(self):
         return "Placeholder({})".format(self.body)
 
 
+HtmlSanitizers = Literal["strip", "escape", "passthrough", "strip_dvla_markup"]
+
+
 class Field:
+    # this needs to be made conditional so it works in the (((colour))) -> (blue) case
```

Comment:
With the regular expression suggested below that reuses yours, we could add this comment to make sure it is properly explained:

```suggestion
    # this needs to be made conditional so it works in the (((colour))) -> (blue) case
    # Deconstructed regular expression segments in order:
    # * First segment: opening ((,
    # * negative lookahead assertion to enforce consumption of late parenthesis and not early ones,
    # * body of placeholder - potentially standard or conditional,
    # * closing ))
```